README.md

```diff
@@ -0,0 +1,84 @@
+# clatd - CLAT, SIIT-DC Edge Relay and IPv6-only for Linux
+
+**clatd** implements the CLAT component of the 464XLAT network architecture
+specified in *RFC 6877*. It allows an IPv6-only host to have IPv4 connectivity
+that is translated to IPv6 before being routed to an upstream PLAT (which is
+typically a Stateful NAT64 operated by the ISP) and there translated back to
+IPv4 before being routed to the IPv4 internet. This is especially useful when
+local applications on the host requires actual IPv4 connectivity or cannot
+make use of DNS64 (for example because they use legacy AF_INET socket calls,
+or if they are simply not using DNS64).
+
+**clatd** may also be used to implement an SIIT-DC Edge Relay as described in
+*RFC 7756*. In this scenario, the PLAT is in reality a SIIT-DC Border Relay
+(see *RFC 7755*) instead of a Stateful NAT64 (see *RFC 6146*). When used as a
+SIIT-DC Edge Relay, you will probably want to manually configure the settings
+`clat-v4-addr`, `clat-v6-addr`, and `plat-prefix` to mirror the SIIT-DC
+Border Relay's configuration.
+
+It relies either on the software package TAYGA by Nathan Lutchansky or on the
+kernel module nat46 by Andrew Yourtchenko for the actual translation of packets
+between IPv4 and IPv6 (*RFC 6145*) TAYGA may be downloaded from its home page
+at <http://www.litech.org/tayga/>, nat46 from its repository at
+<https://github.com/ayourtch/nat46>.
+
+## Installing
+
+clatd is available in the following distributions:
+
+- [Arch (AUR)](https://aur.archlinux.org/packages/clatd-git)
+- [Debian (sid)](https://packages.debian.org/search?keywords=clatd)
+- [Fedora](https://packages.fedoraproject.org/pkgs/clatd/)
+- [Nix](https://search.nixos.org/packages?show=clatd&type=packages&query=clatd)
+- [OpenSUSE](https://software.opensuse.org/package/clatd)
+
+Probably others also. If clatd is not available from your distribution or
+you would like to test the bleeding-edge version of **clatd** from git use
+the following commands:
+
+    $ git clone https://github.com/toreanderson/clatd
+    $ sudo make -C clatd installdeps start
+
+This will install **clatd**, dependencies, systemd services, NetworkManager
+dispatcher integration scripts on your system, enable and start the
+clatd.service. Really only recommended for testing.
```

Comment:
Why *«Really only recommended for testing»*?

`make install` is the supported way of installing clatd that should work anywhere. While users are of course free to use distro packages instead, they are often not available or outdated.

(Testing is better done by running clatd directly from the Git repo, there is no need to install it anywhere.)